### PR TITLE
turn ”react/prefer-es6-class" on as an error

### PR DESCRIPTION
**Pull request by yhan64**
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prefer-es6-class.md

### CODE REVIEW

**Comment by mpaulucci** [comment]
👍

File: es5/fix.js [hunk 1]
```diff
@@ -29,7 +29,8 @@ var es5Options = {
       sourceType: 'script'
     },
     rules: {
-      'react/display-name': 2
+      'react/display-name': 2,
+      "react/prefer-es6-class": 0,
```
Comment (yhan64) [inline es5/fix.js]: @mpaulucci Does this work?

Comment (mpaulucci) [inline es5/fix.js]: it should! 👍

**Comment by indexzero** [comment]
Merging this so that we can publish a minor before we formally deprecate after #40.

**Pull request merged**